tests/test_monitor_system.py

```diff
@@ -0,0 +1,205 @@
+import time
+from pathlib import Path
+import dpath
+
+from dvclive import Live
+from dvclive.monitor_system import (
+    CPUMonitor,
+    METRIC_CPU_COUNT,
+    METRIC_CPU_USAGE_PERCENT,
+    METRIC_CPU_PARALLELIZATION_PERCENT,
+    METRIC_RAM_USAGE_PERCENT,
+    METRIC_RAM_USAGE_GB,
+    METRIC_RAM_TOTAL_GB,
+    METRIC_DISK_USAGE_PERCENT,
+    METRIC_DISK_USAGE_GB,
+    METRIC_DISK_TOTAL_GB,
+)
+from dvclive.utils import parse_metrics
+
+from voluptuous import Union
+from pytest_voluptuous import S
+
+
+def mock_psutil(mocker):
+    mocker.patch(
+        "dvclive.monitor_system.psutil.cpu_percent",
+        return_value=[10, 20, 30, 40, 50, 60],
+    )
+    mocker.patch("dvclive.monitor_system.psutil.cpu_count", return_value=6)
+
+    mocked_virtual_memory = mocker.MagicMock()
+    mocked_virtual_memory.percent = 20
+    mocked_virtual_memory.total = 4 * 1024**3
+
+    mocked_disk_usage = mocker.MagicMock()
+    mocked_disk_usage.used = 16 * 1024**3
+    mocked_disk_usage.percent = 50
+    mocked_disk_usage.total = 32 * 1024**3
+
+    mocking_dict = {
+        "virtual_memory": mocked_virtual_memory,
+        "disk_usage": mocked_disk_usage,
+    }
+    for function_name, return_value in mocking_dict.items():
+        mocker.patch(
+            f"dvclive.monitor_system.psutil.{function_name}",
+            return_value=return_value,
+        )
+
+
+def test_get_cpus_metrics_mocker(tmp_dir, mocker):
+    mock_psutil(mocker)
+    with Live(
+        tmp_dir,
+        save_dvc_exp=False,
+        monitor_system=False,
+    ) as live:
+        monitor = CPUMonitor(disks_to_monitor={"main": "/", "home": "/"})
+        monitor(live)
+        metrics = monitor._get_metrics()
```

Comment:
what is the purpose of this test? why a single one that runs Live at the bottom not enough?

Reply:
I wanted to ensure that the mock objects covered all the fields used in the cpu_monitoring. But after your comment, I realized this test was useless since it only tests behavior that is defined in the unit tests. I removed it.